frontend/src/components/reminder/ReminderTab.js

```diff
@@ -48,14 +52,30 @@ class ReminderTab extends React.Component {
     // Adds entry to list
     handleSubmit = (item) => {
         console.log("handleSubmit");
-        console.log("item:", item);
+        // console.log("item:", item);
+        const oldList = [...this.state.entryList];
 
         this.toggle();
 
         this.state.entryList.push(item);
-        console.log("entryList:",this.state.entryList);
+        // console.log("entryList:",this.state.entryList);
+
+        if (this.state.entryList.length == 1) {
+            axios.post("http://localhost:8000/api/reminders/list", [item]);

```

Comment:
you could replace `http://localhost:8000/` with `process.env.REACT_APP_SERVER_BASE_URL`